### PR TITLE
doc: update instruction on how to release tns-core-modules

### PR DESCRIPTION
**Pull request by SvetoslavTsenov**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -215,39 +215,91 @@ If you want to contribute, but you are not sure where to start - look for [issue
...
+```
+git cherry-pick commit-sha
+git push 
 ```
+5. Merge the PR into release branch.
 
-2. Add the following to your `.npmrc`:
+6. Create a PR to cut the release:
+```
+git checkout -b username/release-version
```
Comment (vakrilov) [inline CONTRIBUTING.md]: Better remove the `username/` from `"username/release-version"`. Same applies to the following commands using this branch.

**Comment by SvetoslavTsenov** [comment]
test --ignore

**Comment by SvetoslavTsenov** [comment]
test --ignore

**Comment by SvetoslavTsenov** [comment]
test --ignore

**Comment by SvetoslavTsenov** [comment]
test --ignore

**Pull request merged**